### DISCUSSION TITLE
unable to add

### DISCUSSION THREAD

**Discussion by Pychangeric** [community/community#51985] [Actions]
### Select Topic Area

Question

### Body

when I try to git push it is giving this error how do I go about it     Enumerating objects: 13, done.
Counting objects: 100% (11/11), done.
Delta compression using up to 4 threads
Compressing objects: 100% (7/7), done.
send-pack: unexpected disconnect while reading sideband packet
Connection to github.com closed by remote host.
fatal: the remote end hung up unexpectedly

**Comment by meppu** [comment]
Did you tried to push a big file? It may also because of your network stability. If the network is too slow, then it might disconnect the connection unexpectedly. If you think your internet is good enough, navigate into project directory and try updating post buffer size (500MB example):

```
git config http.postBuffer 524288000
```

If you use ssh instead of http, replace `http.postBuffer` to `ssh.postBuffer`:

```
git config ssh.postBuffer 524288000
```
